configure.ac

```diff
@@ -3115,6 +3115,73 @@ else
   TCLTK_LIBS="$with_tcltk_libs"
 fi
 
+# check for _gdbmmodulec dependencies
+AC_CHECK_HEADERS([gdbm.h], [
+  LIBS_SAVE=$LIBS
+  AC_CHECK_LIB([gdbm], [gdbm_open])
+  LIBS=$LIBS_SAVE
+])
+
+# check for _dbmmodule.c dependencies
+AC_CHECK_HEADERS([ndbm.h], [
+  LIBS_SAVE="$LIBS"
+  AC_CHECK_LIB([ndbm], [dbm_open])
+  LIBS="$LIBS_SAVE"
+  AC_CHECK_LIB([gdbm_compat], [dbm_open])
+  LIBS="$LIBS_SAVE"
+])
+
+# "gdbm-ndbm.h" and "gdbm/ndbm.h" are both normalized to "gdbm_ndbm_h"
+AS_UNSET([ac_cv_header_gdbm_ndbm_h])
```

Comment:
Why do we need to unset `ac_cv_header_gdbm_ndbm_h` all the time?

Reply:
``AC_CHECK_HEADER`` always caches the result in ``ac_cv_header_$name``. Both ``gdbm-ndbm.h`` and ``gdbm/ndbm.h`` are normalized to ``gdbm_ndbm_h``. It's super annoying but it's the only way to check for both header files without a false positive cache entry.

Reply:
All right. Could you make the comment a little bit more explicit about that? :)